sdk/eventhub/Azure.Messaging.EventHubs/tests/Diagnostics/DiagnosticsTests.cs

```diff
@@ -247,7 +247,7 @@ public async Task CheckpointManagerCreatesScope()
             var context = new PartitionContext(eventHubName, "partition");
             var data = new EventData(new byte[0], sequenceNumber: 0, offset: 0);
 
-            var processor = new EventProcessorClient("cg", new InMemoryPartitionManager(), fakeConnection, null);
+            var processor = new EventProcessorClient("cg", new MockCheckPointStorage(), fakeConnection, null);
```

Comment:
@kinelski -  Do we need an actual mock implementation here, or can we get by with a stub `Mock<PartitionManager>` do you think?